src/ui/features/referral-program/shared/PremiumTrialBanner/PremiumTrialBanner.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import GiftIcon from 'jsx:src/ui/assets/gift.svg';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import { GradientBorder } from '../GradientBorder';
+import * as styles from './styles.module.css';
+
+function GradientText({ children }: { children: React.ReactNode }) {
+  return <div className={styles.gradientText}>{children}</div>;
+}
+
+export function PremiumTrialBanner({ width }: { width: number }) {
```

Comment:
A small problem with this banner layout in narrow view
![Zerts Screenshot 2024-10-22 at 12 46@2x](https://github.com/user-attachments/assets/7ce54cb7-fea6-42d7-bfa2-801ac3024523)

Reply:
Should be fixed ✅